src/vendored/geoarrow/geoarrow.c

```diff
@@ -8607,10 +8607,11 @@ GeoArrowErrorCode GeoArrowWKTWriterInit(struct GeoArrowWKTWriter* writer) {
   ArrowBitmapInit(&private->validity);
   ArrowBufferInit(&private->offsets);
   ArrowBufferInit(&private->values);
-  writer->precision = 16;
-  private->precision = 16;
-  writer->use_flat_multipoint = 1;
-  private->use_flat_multipoint = 1;
+  // TODO should be changed upstream to make this configurable
+  writer->precision = 6;
+  private->precision = 6;
```

Comment:
@paleolimbot what would you recommend here? 

Pass that as an extra param to GeoArrowArrayWriterInitFromSchema, which can then pass it down here?

Or a setter after the object has been initialized? (I suppose that is more backwards compatible)

Reply:
Thanks!